env.example.yml

```diff
@@ -1,7 +1,8 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
-GEOCODE_EARTH_API_KEY: INSERT GEOCODE.EARTH API KEY HERE
-GEOCODE_EARTH_URL: https://api.geocode.earth/v1
+GEOCODER_API_KEY: INSERT HERE API KEY HERE
```

Comment:
Change comment to make the geocoder generic, or change to mention other supported geocoders.